x-pack/plugins/enterprise_search/server/lib/connectors/update_connector_service_type.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from '@kbn/core/server';
+import { i18n } from '@kbn/i18n';
+
+import { CONNECTORS_INDEX } from '../..';
+
+import { ConnectorDocument } from '../../../common/types/connectors';
+
+export const updateConnectorServiceType = async (
+  client: IScopedClusterClient,
+  connectorId: string,
+  serviceType: string
+) => {
+  const connectorResult = await client.asCurrentUser.get<ConnectorDocument>({
```

Comment:
Do we want to wrap this in a try catch or nah?

Reply:
I think its fine to just bubble this up